tests/verify_membackend.erl

```diff
@@ -252,35 +255,55 @@ get_remote_vnode_pid(Node) ->
                                all_vnodes, [riak_kv_vnode]),
     VNode.
 
-%% this is silly fragile
+%% @doc Crack open the VNode state record to find
+%% the hidden number of used bytes within the
+%% riak_kv_memory_backend:state record.
+-spec parse_regular_state_fun(integer()) -> fun().
+parse_regular_state_fun(Offset) ->
+    fun(X) ->
+        element(Offset, element(4, element(2, X)))
+    end.
+
+%% @doc Crack open the VNode state record to find
+%% the hidden number of used bytes within the
+%% riak_kv_memory_backend:state record for a multi-
+%% backend
+-spec parse_multi_state_fun(integer()) -> fun().
+parse_multi_state_fun(Offset) ->
+    fun(X) ->
+        element(
+            3, lists:nth(
+                1, element(
+                    2, element(
+                        Offset, element(
+                            4, element(2, X))))))
+    end.
+
+%% this is silly fragile and only works for Riak 2.0+
```

Comment:
Yea, the parsing of state records like that is pretty horrible - I guess for 2.0/2.1 we could just add the "private tell me your memory usage" function, and keep the horribleness for 1.4 until we officially kill it. That said, I'm OK with this fix for now if the test runs for 2.1, and we can continue to clean it up post-2.1.2 release as part of the "flappy test rewrite" work.